asv.conf.json

```
@@ -1,32 +1,65 @@
 {
+    "version": 1,
+
     // The versions of Python to test against
-    "pythons": ["3.6"],
+    "pythons": ["3.8"], // Python 3.7 included below with extra cached_property dependency
 
-    "environment_type": "conda",
-    "conda_channels": ["conda-forge", "defaults"],
+    // "environment_type": "virtualenv",
+    "environment_type": "virtualenv",
 
     // The matrix of dependencies to test.  Each key is the name of a
     // package (in PyPI) and the values are version numbers.  An empty
-    // list indicates to just test against the default (latest)
+    // string indicates to just test against the default (latest)
     // version.
     "matrix": {
-        "numpy": ["1.14"],
-        "numba": ["0.39"]
+        "numpy": "",
+        "numba": "",
```

What about fixing the versions? That way we control when do we upgrade the benchmarks and notice when a new dependency introduces a performance change. I propose:

```suggestion
        "numpy": ["1.17.5"],
        "numba": ["0.47.0"],
        "llvmlite": ["0.31.0"]
```

these versions are the oldest ones with Python 3.8 wheels, I think.